Lib/test/test_tempfile.py

```diff
@@ -803,6 +804,33 @@ def test_mode(self):
         finally:
             os.rmdir(dir)
 
+    @unittest.skipUnless(os.name == "nt", "Only on Windows.")
+    def test_mode_win32(self):
+        # Use icacls.exe to extract the users with some level of access
```

Comment:
This doesn't call `os.mkdir` directly. I'm assuming it gets called somewhere in tempfile, but wouldn't it be better to test this behavior in a test that calls `os.mkdir` directly?